skimage/transform/_geometric.py

```diff
@@ -1294,7 +1296,7 @@ def warp(image, inverse_map=None, map_args={}, output_shape=None, order=1,
     >>> warped = warp(cube, coords)
 
     """
-
+    mode = _mode_deprecations(mode)
```

Comment:
Further below, we fall back to ndimage. The modes need to be wrapped there as well.